flake.nix

```diff
@@ -2,7 +2,7 @@
   description = "Neovim derivation";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/55d15ad12a74eb7d4646254e13638ad0c4128776";
```

Comment:
issue: There is no need for this. The input is locked in the `flake.lock`.
Whenever I run `nix flake update`, I check to make sure the `wrapNeovimUnstable` function still works.
If it doesn't, I update this template accordingly.

Reply:
Ah, ok then. I thought you was about locking it forever

Reply:
I dropped the commit, and now the inputs are intact